pkg/crc/config/viper_config.go

```diff
@@ -57,7 +57,6 @@ func New(configFile, envPrefix string) (*Config, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Error reading configuration file '%s': %v", configFile, err)
 	}
-	v.WatchConfig()
```

Comment:
This is still needed for the daemon..

Reply:
Thanks for catching that! I added a test https://github.com/code-ready/crc/pull/1460/commits/4eef250b7ff30b3123a88daf1f51b4c2a78a9b58